samples/bluetooth/mesh/dfu/distributor/prj.conf

```diff
@@ -30,6 +30,7 @@ CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_SETTINGS=y
 CONFIG_BT_ECC=y
 CONFIG_BT_TINYCRYPT_ECC=y
+CONFIG_BT_BUF_CMD_TX_COUNT=4
```

Comment:
Should this also be mentioned in documentation? 

Also, why only distributor sample? It seems like this setting is needed in all samples supporting relay, friendship, and GATT.

Reply:
> Should this also be mentioned in documentation as a known issue?
> 
This is not a direct mesh issue, that is why I did not considered this as a know issue for mesh. Most "hosts'" known issue.
> Also, why only distributor sample? It seems like this setting is needed in all samples supporting relay, friendship, and GATT.

Distributor has SMP service in addition to Mesh ADV and Mesh GATT that's, that's why for other samples the default buffer length is sufficient.